File: 1_4/1task.js
let N = 26;
if (N > 26) {
    console.log("Incorrect data");
} 
else {
    for (let i = N; i > 0; i--) {
        let Str1 = "";
        for (let k = N; k >= i; k--) {
            let Str = String.fromCharCode('A'.charCodeAt(0) + k - 1);
            Str1 = Str1 + Str + " ";
        }
        console.log(Str1 + " ");
    }
}

File: 1_4/1task.cpp
#include <iostream>
using namespace std;
int main() {
    int N = 4;
    if (N>26) {
        cout << "Incorrect data" << endl;
    }
    else {
        for (int i = N ; i>0; i--) {
            for (int k=N ; k>=i ; k--) {
                char Str = 'A' + k - 1;
                cout << Str << " ";
            }
            cout << endl;
        }
    }
}